src/layout/layout.js

```diff
@@ -328,6 +420,10 @@
         this.tabBar_ = this.header_.querySelector('.' + this.CssClasses_.TAB_BAR);
       }
 
+      if (this.drawer_) {
+        this.drawerMenuItems_ = this.drawer_.querySelectorAll('.mdl-navigation__link');
```

Comment:
Thats a good point.  With the current docs for MDL lite, how would these elements be indicated with existing classes (or would they)?  The config for nav layouts doesnt really address this.